doc/webui/index.rst

```diff
@@ -0,0 +1,124 @@
+.. _webui:
+
+WebUI
+=========
+
+privacyIDEA comes with a web-based user interface which is used to manage and configure
+the privacyIDEA server. With the correct policies, it is also used a self-service portal
+for the average user. This section gives an overview on the interface and links the
+respective sections in the documentation.
+
+.. note:: For productive environments, it is absolutely necessary to secure the connection with HTTPS.
+   PrivacyIDEA only comes with a self-signed certificate, which serves for testing environments.
+
+Tokens
+-------
+
+.. index:: tokensview
+
+The administrator can see all the tokens of all realms he is allowed to manage in the
+tokenview. Each token can be located in several realms and be assigned to one
+user. The administrator can see all the details of the token.
+
+.. figure:: images/tokensview.png
+   :width: 500
+
+   *Tokens overview*
+
+The administrator can click on one token, to show more details of this token
+and to perform actions on this token. Read on in :ref:`token_details`.
+
+
+Users
+-------
+
+The administrator can see all users fetched by :ref:`useridresolvers` located in
+:ref:`realms` he is allowed to manage.
+
+.. note:: Users are only visible, if the useridresolver is located
+   within a realm. If you only define a useridresolver but no realm,
+   you will not be able to see the users!
+
+You can select one of the realms in the left drop down box. The administrator
+will only see the realms in the drop down box, that he is allowed to manage.
+
+
+.. figure:: images/usersview.png
+   :width: 500
+
+   *The Users view list all users in a realm.*
+
+The list shows the users from the select realm. The username, surname,
+given name, email and phone are filled according to the definition of
+the useridresolver.
+
+Even if a realm contains several useridresolvers all users from all
+resolvers within this realm are displayed.
+
+Read about the functionality of the users view in the following sections.
+
+.. toctree::
+   :maxdepth: 1
+
+   user_details
+   manage_users
+
+
+Machines
+-------
+
+.. todo:: Short sentence about the purpose of machine-resolving.
```

Comment:
Machines are only necessary if you plan special use cases like managing SSH keys or doing offline OTP.
In most cases there is no need to manage machines.

Reply:
done

Reply:
added a sentence.